polkadot/node/network/availability-distribution/tests/availability-distribution-regression-bench.rs

```diff
@@ -57,13 +58,13 @@ fn main() -> Result<(), String> {
 	println!("{}", usage);
 
 	messages.extend(usage.check_network_usage(&[
-		("Received from peers", 443.333, 0.05),
-		("Sent to peers", 21818.555, 0.05),
+		("Received from peers", 433.333, 0.05),
+		("Sent to peers", 18480.000, 0.05),
```

Comment:
The amount of data sent/received should always be the same if we don't randomize the PoV sizes, or we use the same seed.